### PR TITLE
Feature 11494

### PR DESCRIPTION
**Pull request by Kevin-Wei**
right target this time

### CODE REVIEW

File: app/controllers/resources/v1/policies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module Resources
+  module V1
+    class PoliciesController < ApplicationController
+      skip_before_filter :authenticate_me!
+
+      def show
+        @policy = Policy.where("eg_id" => params[:id]).first
```
Comment (TreyE) [inline app/controllers/resources/v1/policies_controller.rb]: Sadly, you are going to need to look this up by the actual 'mongoid' of the policy, as eg_id isn't unique.

File: app/views/policies/show.xml.haml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        %id
+          %id= "urn:openhbx:hbx:dc0:resources:v1:person:hbx_id##{enrollee.person.authority_member.hbx_member_id}"
+        = render "people/person.xml.haml", :person => enrollee.person
+        %person_demographics
+          %ssn= enrollee.person.authority_member.ssn
+          %sex= "urn:openhbx:terms:v1:gender##{enrollee.person.authority_member.gender}"
+          %birth_date= simple_date_for(enrollee.person.authority_member.dob)
+      %is_subscriber= enrollee.subscriber?
+      %benefit
+        %premium_amount= enrollee.pre_amt
```
Comment (TreyE) [inline app/views/policies/show.xml.haml]: Double check the formatting on this bad boy, make sure it comes out to cents.

Comment (Kevin-Wei) [inline app/views/policies/show.xml.haml]: good looking out

File: app/views/policies/show.xml.haml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        %begin_date= simple_date_for(enrollee.coverage_start)
+        - if !enrollee.coverage_end.blank?
+          %end_date= simple_date_for(enrollee.coverage_end)
+        - if enrollee.cp_id.present?
+          %carrier_assigned_policy_id= enrollee.cp_id
+        - if enrollee.c_id.present?
+          %carrier_assigned_enrollee_id= enrollee.c_id
+  - if policy.has_responsible_person?
+    %responsible_party
+      = render "people/person.xml.haml", :person => policy.responsible_person
```
Comment (TreyE) [inline app/views/policies/show.xml.haml]: Looks like the responsible party stuff is a little nastier than simply plugging in the person.  In order to get the right ID element you are going to need to do a little more.  You will be able to use 90% of the person template, but will probably needs some special stuff around the ID tag.  I would recommend a separate partial.

**Comment by TreyE** [comment]
Going to merge this as a 3.5 branch.

**Comment by Kevin-Wei** [comment]
addressed via https://github.com/dchbx/enroll/pull/39

**Pull request closed without merging**